app/cafe.py

```
@@ -0,0 +1,22 @@
+from datetime import datetime
+
+from app.errors import (OutdatedVaccineError,
+                        NotWearingMaskError, NotVaccinatedError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Not Vaccinated")
+
+        if visitor["vaccine"]["expiration_date"] < datetime.today().date():
+            raise OutdatedVaccineError("Outdated Vaccine")
+
```

This line accesses `visitor["wearing_a_mask"]` directly. The task states the mask error should be raised when the key is present and explicitly False. Direct indexing will raise a KeyError if the key is missing. Change this to use `.get()` and only raise when it is explicitly False, for example: `if visitor.get("wearing_a_mask") is False:`. Also the associated `NotWearingMaskError` message should be descriptive and include the visitor's name per checklist item #3.